### DISCUSSION TITLE
Simulation Freezes

### DISCUSSION THREAD

**Discussion by danielb-28** [thliebig/openEMS-Project#63] [Q&A Usage] [answered]
I have been trying to simulate a pair of coupled MSL. When using a Gaussian pulse excitation under some conditions the electric field will freeze when the excitation is near its peak. I've noted that this is influenced by the position of the excitation plane: if the plane is too close to the boundary the problem will occur. 

Is there a simple answer to what causes this behavior or it can be caused by multiple reasons? If so, how can I debug a trace the cause of the problem?

**Comment by 0xCoto** [comment, marked as answer]
> if the plane is too close to the boundary the problem will occur.

In general, this is usually not a good idea, I'd suggest keeping a gap of at least λ/4 from the ends of the structure in each direction with ~8 PML cells (if you're using PML).

**Reply by thliebig** [reply]
I agree, this sounds like you pumped some DC into the structure and/or PML...  Think of it like a static charge that cannot go anywhere...
